<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Vernamco</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTextEdit" name="TextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="SaveAs"/>
    <addaction name="Close"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="EncryptionMenu">
    <property name="title">
     <string>Encryption</string>
    </property>
    <addaction name="MaskEncryption"/>
    <addaction name="SymbolReplacement"/>
   </widget>
   <widget class="QMenu" name="DecryptionMenu">
    <property name="title">
     <string>Decryption</string>
    </property>
    <addaction name="MaskDecryption"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EncryptionMenu"/>
   <addaction name="DecryptionMenu"/>
  </widget>
  <action name="Open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="MaskEncryption">
   <property name="text">
    <string>Mask</string>
   </property>
  </action>
  <action name="MaskDecryption">
   <property name="text">
    <string>Mask</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="Close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="SymbolReplacement">
   <property name="text">
    <string>Symbol Replacement</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
